build.gradle.kts

```diff
@@ -100,38 +100,6 @@ buildTypes {
         tasks("vfsRetentionIntegTest")
     }
 
-    create("performanceTests") {
```

Comment:
These build types are removed because they're unnecessary. It also cause problems that `--baselines` doesn't work for `performanceTests` tasks.